afs/afs-storage-api/src/main/java/com/powsybl/afs/storage/EventsStore.java

```diff
@@ -0,0 +1,37 @@
+/**
+ * Copyright (c) 2019, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.afs.storage;
+
+import com.powsybl.afs.storage.events.AppStorageListener;
+import com.powsybl.afs.storage.events.NodeEvent;
+
+/**
+ * @author Chamseddine Benhamed <chamseddine.benhamed at rte-france.com>
+ */
+public interface EventsStore {
```

Comment:
Actually it looks more like an event queue than an event store (the interface says nothing about persistence / storage of the vents).
I think we should rename it to `EventsQueue` or `EventQueue`.

Reply:
From a broader point of view, I wonder if the events queue should be tightly coupled to AFS (as for services), since an application may want to broadcast events and messages not directly related to AFS.

Reply:
> Actually it looks more like an event queue than an event store (the interface says nothing about persistence / storage of the vents).
> I think we should rename it to `EventsQueue` or `EventQueue`.

I propose EventsBus what do you think ?

Reply:
You're right it's probably better :)
"queue" may sound like there is only 1 consumer/subscriber.

@geofjamg any opinion about this naming topic ?

Reply:
@sylvlecl  Exactly, Bus is more expressive than Queue .

Reply:
> tightly
Alright, maybe we have to refactor that in a separate PR, to build a system that supports new events Type (because for kafka impl we have to create serializers and deserializers for each event Type).

Reply:
> From a broader point of view, I wonder if the events queue should be tightly coupled to AFS (as for services), since an application may want to broadcast events and messages not directly related to AFS.

Alright, maybe we have to refactor that in a separate PR, to build a system that supports new events Type (because for kafka impl we have to create serializers and deserializers for each event Type).